civihr_default_theme/scss/components/_manager_dashboard.scss

```diff
@@ -0,0 +1,52 @@
+.chr_manager_dashboard__header {
+  margin: 0;
+
+  .chr_custom-select {
+    margin-right: 20px;
+    padding: 0;
+    width: 250px;
+  }
+
+  .panel-title {
+    margin-top: 5px;
+    display: inline-block;
+  }
+
+  > div {
+    display: inline-block;
+    float: right;
+  }
+}
+
+.chr_manager_dashboard__filter {
+  margin-top: 6px;
+  display: inline-block;
+
+  .fa {
+    font-size: 0.6em;
+    vertical-align: middle;
+    margin-right: 5px;
+  }
+}
+
+.chr_manager_dashboard__sub-header {
+  padding-top: 15px;
+
+  > div {
+    display: inline-block;
+    margin: 0 10px;
+    width: calc(16.4% - 20px);
+    vertical-align: top;
+  }
+}
+
+.chr_manager_dashboard__panel_body {
+  margin: 0;
+  padding: 0;
+
+  [class^="col-"],
+  [class*=" col-"] {
```

Comment:
To be honest, i am not really sure of the implications of changing this, I read the documentation and it seems that it might work. But Bootstrap too is using like this ( `[class^="col-"],[class*=" col-"]`), so I guess its better to keep like this to keep consistency?